uid2_client/encryption.py

```diff
@@ -191,12 +191,12 @@ def _encrypt_token(uid2, identity_scope, master_key, site_key, site_id, now, tok
 
 
 
-def encrypt(uid2, indentity_scope, keys, keyset_id=None, **kwargs):
+def encrypt(uid2, identity_scope, keys, keyset_id=None, **kwargs):
     """ Encrypt an uid2 into a sharing token
 
     Args:
         uid2: the uid2 to be encrypted
-        indentity_scope (IdentityScope): If the key will be uid2 or euid2
+        identity_scope (IdentityScope): If the key will be uid2 or euid2
```

Comment:
Sunny a question here. If these are plain language values, UID2 or EUID. If they are code values, not sure still if it's euid2? I never saw euid2 mentioned, in code or copy. Just checking. Probably it's fine.

Reply:
actually it doeesn't quite make sense to say 

_If the key will be uid2 or euid_

as Identity Scope defines whether we want to create a token for EUID or UID2 so it's not really about the key is uid2/euid or not. But then in the code we says the arg uid2 as _"the uid2 to be encrypted"_. Then if we change that it will be a can of worms.... @genwhittTTD would you have any suggestion how we could rephrasse this?

Reply:
i have changed the wording as discussed with @genwhittTTD